lib/double_entry/reporting/aggregate.rb

```diff
@@ -25,7 +25,7 @@ def amount(force_recalculation = false)
         end
       end
 
-      def formatted_amount(value = amount())
+      def formatted_amount(value = amount)
```

Comment:
I don't understand what is circular about the statement.
If `value` is not provided, it calls the `amount` method.
`amount` is not being shadowed here.